tests/test_server.py

```diff
@@ -120,14 +137,59 @@ def test_rtm_reconnect(server, rtm_start_fixture):
             json=rtm_start_fixture
         )
 
-        server.rtm_connect(use_rtm_start=False)
+        server.rtm_connect(auto_reconnect=True, reconnect=True, use_rtm_start=False)
+
+        for call in rsps.calls:
+            assert call.request.url in [
+                "https://slack.com/api/rtm.connect"
+            ]
+
+
+def test_max_rtm_reconnect_timeout_recently_connected(server, rtm_start_fixture):
+    with responses.RequestsMock() as rsps:
+        rsps.add(
+            responses.POST,
+            "https://slack.com/api/rtm.connect",
+            status=200,
+            json=rtm_start_fixture
+        )
+
+        server.reconnect_attempt = 1
+        server.last_connected_at = time.time() - 180
+        server.rtm_connect(auto_reconnect=True, reconnect=True, use_rtm_start=False)
+
+        for call in rsps.calls:
+            assert call.request.url in [
```

Comment:
can you explain how this test works? i don't see how the timeout could have been hit.

Reply:
Added more context to the test

Reply:
not sure i understand. what i see is connecting when the `last_connected_at` time is now will immediately cause another connection attempt. that sounds like the opposite of what your comment says should happen.